### PR TITLE
Add top level README for GAE Java11 bundled services samples, based on Java8 samples.

### PR DESCRIPTION
**Pull request by ludoch**
Fixes #issue

> It's a good idea to open an issue first for discussion.

- [ ] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)
- [ ] `pom.xml` parent set to latest `shared-configuration`
- [ ] Appropriate changes to README are included in PR
- [ ] API's need to be enabled to test (tell us)
- [ ] Environment Variables need to be set (ask us to set them)
- [ ] **Tests** pass:   `mvn clean verify` **required**
- [ ] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [ ] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [ ] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: appengine-java11-bundled-services/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+1. Open the [Cloud Platform Console][cloud-console].
+1. In the drop-down menu at the top, select **Create a project**.
+1. Give your project a name.
+1. Make a note of the project ID, which might be different from the project
+   name. The project ID is used in commands and in configurations.
+
+[cloud-console]: https://console.cloud.google.com/
+
+
+## Difference between App Engine Java8 and Java11 Bundled Services
```
Comment (lesv) [inline appengine-java11-bundled-services/README.md:40]: `Development differences` ??

File: appengine-java11-bundled-services/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+This sample demonstrates how to use the App Engine Datastore APIs in a Java11 web application on Google App Engine Java11.
+
+- [Documentation][ae-docs]
+- [Code](https://github.com/GoogleCloudPlatform/java-docs-samples/tree/main/appengine-standard-java11-bunded-services/datastore)
+
+### How to change an App Engine Java 8 application to App Engine Java11 bundled services
+
+You can execute the following steps to transform the java8 appengine-web.xml file to a java11 appengine-web.xml file:
+
```
Comment (lesv) [inline appengine-java11-bundled-services/README.md:67]: \`\`\`shell

File: appengine-java11-bundled-services/README.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+     git clone https://github.com/GoogleCloudPlatform/java-docs-samples.git
+	 cd java-docs-samples
+	 cp -pr appengine-java8 /tmp/java11-samples
+	 cd /tmp/java11-samples
+	 # On Linux:
+	 shopt -s globstar dotglob
+     for f in **/appengine-web.xml; do sed -i 's.<runtime>java8</runtime>.<runtime>java11</runtime><app-engine-apis>true</app-engine-apis>.' ${f}; done 
+	 # on MacOS
+     for f in **/appengine-web.xml; do sed -i'' -e 's.<runtime>java8</runtime>.<runtime>java11</runtime><app-engine-apis>true</app-engine-apis>.' ${f}; done 
+
```
Comment (lesv) [inline appengine-java11-bundled-services/README.md:77]: \`\`\`

**Review by averikitsch — COMMENTED** [review]
Does gcloudMode for the app engine plugin still need to be set to beta? https://github.com/GoogleCloudPlatform/app-maven-plugin/blob/master/USER_GUIDE.md#deploy-1

**Comment by ludoch** [comment]
Did not have to use beta, but maybe because I am a googler. Just updated the README to use the gcloudMode beta while we are in Beta.

**Review by lesv — APPROVED** [review]

**Pull request merged**